animatch/EditProfile/EditProfile1.swift

```diff
@@ -0,0 +1,191 @@
+//
+//  EditProfile1.swift
+//  animatch
+//
+//  Created by Ксения Панкратова on 18.11.2024.
+//
+
+import SwiftUI
+
+
+
+struct EditProfile1: View {
+    var body: some View {
+        VStack {
+            ZStack{
+                LinearGradient(gradient: Gradient(colors: [.editProfLightGrad, .editProfDarkGrad]), startPoint: .top, endPoint: .center)
+                    .ignoresSafeArea()
+             
+                VStack(spacing: 0){
+                    
+                    Head()
+                    
+                    Spacer()
+                    
+                    Forms()
+                    
+                    //у полей ввода нижние паддинги 30
+                        .padding(.bottom, 124)
+                    VStack(spacing: 0){
+                        
+                        ZStack{
+                            CustomButton(placeholder: "Сохранить и продолжить")
+                        }
+                        .padding(.bottom, 8)
+                        
+                        Button(action: {
+                            print("Сделать позже")
+                        }) {
+                            Text("Сделать позже")
+                                .underline()
+                                .foregroundColor(.editProfPurple)
+                        }
+                    }.padding(.bottom, 64)
+                    
+                }.ignoresSafeArea()
+                
+            }
+        }
+        
+    }
+}
+
+struct Head: View{
+    var body: some View{
+        ZStack(alignment: .top){
+                
+            UnevenRoundedRectangle(cornerRadii: .init(bottomLeading: 24,bottomTrailing: 24))
+                .fill(Color.editProfPurple)
+                .frame(height: 166)
+            
+                Text("Профиль")
+                    .foregroundStyle(.editProfWhite)
+                    .font(Font.custom("Roboto", size: 20))
+                    .offset(y:55)
+                    .padding(.horizontal, 16)
+
+            ZStack(alignment: .top){
+                Avatar()
+            }.offset(y: 123)
```

Comment:
ZStack тут тоже не нужен

```suggestion
            Avatar()
                .offset(y: 123)
```